packages/internals/src/index.ts

```diff
@@ -59,9 +60,8 @@ export { getEnvPaths } from './utils/getEnvPaths'
 export { version } from './utils/getVersionFromPackageJson'
 export { handlePanic } from './utils/handlePanic'
 export { isCi } from './utils/isCi'
-export { isInteractive } from './utils/isInteractive'
-export { canPrompt } from './utils/canPrompt'
 export { isCurrentBinInstalledGlobally } from './utils/isCurrentBinInstalledGlobally'
+export { isInteractive } from './utils/isInteractive'
```

Comment:
This change is unexpected, looks like a merge went wrong

Reply:
Resolved: the linting did this